runtime/util/filecache.h

```diff
@@ -26,16 +26,12 @@
 #include "j9.h"
 #include "j9port.h"
 
-#define CACHE_BUFFER_SIZE 16*1024
-#define CACHE_BUFFER_NUM 4
-#define INVALID_POSITION -1
-#define UNUSED FALSE
-#define INUSE TRUE
-#ifdef J9ZTPF
-#undef NO_DATA
-#endif
-#define NO_DATA -1
-#define HAS_DATA(index) handle->cacheTable[index].hwm >= 0
+#define J9FILECACHE_BUFFER_SIZE 16*1024
+#define J9FILECACHE_BUFFER_NUM 4
+#define J9FILECACHE_UNUSED FALSE
+#define J9FILECACHE_INUSE TRUE
+#define J9FILECACHE_NO_DATA -1
+#define J9FILECACHE_HAS_DATA(index) handle->cacheTable[index].hwm >= 0
```

Comment:
Please add the missing parentheses for `J9FILECACHE_BUFFER_SIZE`, `J9FILECACHE_NO_DATA` and `J9FILECACHE_HAS_DATA`.

Reply:
Done